yeonjae/programmers-1/수박수박수박.js

```diff
@@ -0,0 +1,19 @@
+function solution(n) {
+  let repeatStr = "";
+
+  for (let i = 1; i<=n; i++) {
+    (i % 2 === 0) ? (repeatStr += "박") : (repeatStr += "수");
+  }
+  
+  return repeatStr;
+}
```

Comment:
```suggestion
function solution(n) {
  let repeatedStr = "";

  for (let i = 1; i<=n; i++) {
    (i % 2 === 0) ? (repeatedStr += "박") : (repeatedStr += "수");
  }
  
  return repeatedStr;
}
```
- repeat 동사보다는 repeatedStr로 변수 잡는게 좋을 것 같습니다~!

Reply:
넵, 감사합니다 리뷰 확인했습니다😁😁